sdk/php/dagger.json

```diff
@@ -1,18 +1,6 @@
 {
   "name": "php-sdk",
   "sdk": "go",
-  "include": [
-    "generated/",
-    "src/",
-    "scripts/",
-    "composer.json",
-    "composer.lock",
-    "LICENSE",
-    "README.md"
-  ],
-  "exclude": [
-    "vendor/"
-  ],
```

Comment:
Don't remove this exclude. The Go SDK adds `**/vendor` to includes, but it's meant for Go modules. However, PHP has a heavy `vendor` and will be included by default without this.